src/util/handlebars/handlebar-helpers.ts

```diff
@@ -4,9 +4,13 @@ import { isPlainObject } from "lodash";
 export const concatStringHelper = (...strings: string[]) => strings.filter((arg: unknown) => typeof arg !== "object").join(" ");
 export const toLowercaseHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().toLowerCase() || "";
 export const replaceSpaceWithHyphenHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().replace(/ /g, "-") || "";
+export const compareOr = (...args: unknown[]) => args.filter((arg: unknown) => typeof arg !== "object").some(Boolean);
```

Comment:
This needs to be renamed to be more explicit. Because helpers are typically 1-lines, the name should tell us what this is doing without us needing to read the code. Also, all helper functions end with `Helper`

Reply:
Variable renamed to `logicalOrHelper` and `logicalAndHelper`.